kroxylicious-test-tools/src/main/java/io/kroxylicious/test/tester/DefaultKroxyliciousTester.java

```diff
@@ -110,8 +110,13 @@ public <U, V> Consumer<U, V> consumer(Serde<U> keySerde, Serde<V> valueSerde, Ma
     }
 
     @Override
-    public KafkaClient singleRequestClient() {
-        return clients().singleRequestClient();
+    public KafkaClient mockRequestClient() {
```

Comment:
I wonder if the naming is misleading (the MockServer is a mock broker, but the client is doing real client work). Maybe `lowLevelClient`, `simpleClient`, `testClient`?

Reply:
yeah @robobario is right. I don't think calling it a mock client is helpful

Reply:
Yes, you are both right.  I'll go with simpleTestClient.